.github/workflows/ci.yml

```
@@ -0,0 +1,14 @@
+name: CI
+
+on: push
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install modules
+        run: yarn
+
+      - name: Run eslint
+        run: yarn lint

```

como a gente ta usando package-lock.json, tem que usar o npm

```suggestion
      - name: Install modules
        run: npm ci

      - name: Run eslint
        run: npm run lint

```